docs/ibd.md

```diff
@@ -0,0 +1,201 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.12
+    jupytext_version: 1.9.1
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+```{currentmodule} tskit
+```
+
+
+(sec_identity)=
+
+# Identity by descent
+
+The {meth}`.TreeSequence.ibd_segments` allows us to compute
+segments of identity by descent.
+
+:::{note}
+This documentation page is preliminary
+:::
+
+## Examples
+
+Let's take a simple tree sequence to illustrate the {meth}`.TreeSequence.ibd_segments`
+method and associated {ref}`sec_python_api_reference_identity`:
+
+```{code-cell}
+:tags: [hide-input]
+
+import tskit
+import io
+from IPython.display import SVG
+
+nodes = io.StringIO(
+    """\
+    id      is_sample   time
+    0       1           0
+    1       1           0
+    2       1           0
+    3       0           1
+    4       0           2
+    5       0           3
+    """
+)
+edges = io.StringIO(
+    """\
+    left    right   parent  child
+    2     10     3       0
+    2     10     3       2
+    0     10     4       1
+    0     2      4       2
+    2     10     4       3
+    0     2      5       0
+    0     2      5       4
+    """
+)
+ts = tskit.load_text(nodes=nodes, edges=edges, strict=False)
+
+SVG(ts.draw_svg())
+```
+
+### Definition
+
+A pair of nodes ``(u, v)`` has an IBD segment with a left and right
+coordinate ``[left, right)`` and ancestral node ``a`` iff the most
+recent common ancestor of the segment ``[left, right)`` in nodes ``u``
+and ``v`` is ``a``, and the segment has been inherited along the same
+genealogical path (ie. it has not been broken by recombination). The
+segments returned are the longest possible ones.
+
+Consider the IBD segments that we get from our example tree sequence:
+
+```{code-cell}
+segs = ts.ibd_segments(store_segments=True)
```

Comment:
I am reliably informed that "seg" is slang for a warty callus (oop north), so I suggest to avoid being off-putting we `s/segs/segments/g` !